README.md

```diff
@@ -1,4 +1,4 @@
-![MASP Logo](https://github.com/metastatedev/masp/blob/main/docs/logo.png)
+![MASP Logo](https://github.com/anoma/masp/blob/main/docs/logo.png)
 
```

Comment:
You can further modify this doc, and remove on line 33 below the mention that `zcash_proofs` and `zcash_primitives` crates were copied, which is no longer the case after this extensive tidying up.